docs/pages/push-notifications/sending-notifications.md

```diff
@@ -21,7 +21,7 @@ When you're ready to send a push notification, take the Expo push token from you
 - [expo-server-sdk-ruby](https://github.com/expo/expo-server-sdk-ruby) for Ruby. Maintained by community developers.
 - [expo-server-sdk-rust](https://github.com/expo/expo-server-sdk-rust) for Rust. Maintained by community developers.
 - [ExpoNotificationsBundle](https://github.com/solvecrew/ExpoNotificationsBundle) for Symfony. Maintained by SolveCrew.
-- [exponent-server-sdk-php](https://github.com/Alymosul/exponent-server-sdk-php) for PHP. Maintained by community developers.
```

Comment:
We could have both PHP packages listed here

Reply:
Updated. Any preference as to same or new line?